src/tools/rbd/action/Qos.cc

```diff
@@ -0,0 +1,120 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
```

Comment:
Call it "config" or "setting" or "property" -- at the end of the day, it's the same thing. We don't need to re-invent the wheel to store a key/value pair.